course-bundle-spec.md

```diff
@@ -0,0 +1,164 @@
+# Qwiklabs Course Bundle Specification
+
+**Version 1**
+
+> This is a DRAFT document. We welcome feedback as this format evolves.
+
+## `qwiklabs.yaml` Structure
+
+Here's a sample `qwiklabs.yaml` file with all nested details removed to make it easier to see the general file structure.
+
+```yml
+entity_type: Course
+schema_version: 1
+
+# Course Attributes
+id: gcp-intro-course
```

Comment:
Ugh...

I know I'm responsible for the `id` field in the lab bundle spec, but I've always been hesitant about keeping an ID field in the course bundle itself instead of having it be externally defined via Alexandria or other publishing system.

@joncass Would you mind chatting with me for 15 minutes in front of a white board today, to talk about the implications of this?

Reply:
We decided to remove `id` from the bundle, and use the folder name as the author-defined `slug`.